packages/fiori/src/IllustratedMessage.js

```diff
@@ -0,0 +1,245 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import { getIllustrationDataSync } from "@ui5/webcomponents-base/dist/asset-registries/Illustration.js";
+
+import { getI18nBundle, fetchI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import IllustratedMessageTemplate from "./generated/templates/IllustratedMessageTemplate.lit.js";
+
+// Styles
+import IllustratedMessageCss from "./generated/themes/IllustratedMessage.css.js";
+
+const ILLUSTRATION_NOT_FOUND = "ILLUSTRATION_NOT_FOUND";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-illustrated-message",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.fiori.IllustratedMessage.prototype */ {
+		/**
+		 * Defines the title of the component.
+		 * <b>Note:</b> Using this property, the default title text of illustration will be overwritten.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @since 1.0.0-rc.15
+		 * @public
+		 */
+		titleText: {
+			type: String,
+		},
+		/**
+		 * Defines the subtitle of the component.
+		 * <b>Note:</b> Using this property, the default subtitle text of illustration will be overwritten.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @since 1.0.0-rc.15
+		 * @public
+		 */
+		subtitleText: {
+			type: String,
+		},
+		/**
+		 * Determinates what is the current media of the component based on its width.
+		 * @private
+		*/
+		media: {
+			type: String,
+		},
+		/**
+		 * Determinates whether illustration is invalid.
+		 * @private
+		*/
+		invalid: {
+			type: Boolean,
+		},
+		/**
+		 * Defines the illustration name that will be displayed in the component.
+		 * <br><br>
+		 * Available illustrations are:
+		 * <ul>
+		 * <li><code>BeforeSearch</code></li>
+		 * <li><code>NoActivities</code></li>
+		 * <li><code>NoData</code></li>
+		 * <li><code>NoEntries</code></li>
+		 * <li><code>NoMail</code></li>
+		 * <li><code>NoNotifications</code></li>
+		 * <li><code>NoSavedItems</code></li>
+		 * <li><code>NoSearchResults</code></li>
+		 * <li><code>NoTasks</code></li>
+		 * <li><code>UnableToLoad</code></li>
+		 * <li><code>UnableToUpload</code></li>
+		 * </ul>
+		 * @type {string}
+		 * @defaultvalue "BeforeSearch"
+		 * @since 1.0.0-rc.15
+		 * @public
+		 */
+		name: {
+			type: String,
+			defaultValue: "BeforeSearch",
```

Comment:
Here we have 11 types, perhaps we can use enumeration of 11 options. Also remind users that they have to import the corresponding image separately. It is not mentioned nether here, nor in the overview.